deploy/charts/cert-manager/templates/controller-config.yaml

```diff
@@ -1,6 +1,6 @@
 {{- if .Values.config -}}
-{{- required ".Values.config.apiVersion must be set !" .Values.config.apiVersion -}}
-{{- required ".Values.config.kind must be set !" .Values.config.kind -}}
+{{- $_ := .Values.config.apiVersion | required ".Values.config.apiVersion must be set !" -}}
+{{- $_ := .Values.config.kind | required ".Values.config.kind must be set !" -}}
```

Comment:
I think this expression is quite hard to understand.
Could we maybe go back to the original code: https://github.com/cert-manager/cert-manager/pull/6360/files#diff-c8ba27b3d1af50639145774c5a78f99fd223062180bf09f8c9616016287e1818L2-L4?